### PR TITLE
T 9

### PR DESCRIPTION
**Pull request by ivanna-skipar**
@wizzardio please check profile page.

### CODE REVIEW

File: client/src/components/layouts/index.js [hunk 1]
```diff
@@ -14,9 +14,17 @@ import TaskList from './userBoard/TaskList';
...
+
 import CardMenuBtn from '../common/CardMenuBtn';
 import Dashboard from '../layouts/userBoard/Dashboard';
 import BoardHeader from '../layouts/userBoard/BoardHeader';
+import MainProfile from './profilePage/MainProfile'
+
+=======
+import CardMenuBtn from '../common/CardMenuBtn'
+import MainProfile from './profilePage/MainProfile'
+>>>>>>> ff04dc59ab6496b1131b020a95d85f779172b55a
```
Comment (wizzardio) [inline client/src/components/layouts/index.js]: conflicts?

**Review by wizzardio — COMMENTED** [review]
I have noticed some git conflicts in your code.

**Comment by ivanna-skipar** [comment]
It is not actually a conflicts. Somebody import two components. Maybe I didn't notice it. I don't see this into my VS Code. How can I fix it?

**Comment by ivanna-skipar** [comment]
I managed that conflict. Can I merge T-9?

**Review by wizzardio — APPROVED** [review]

**Pull request merged**